common/src/main/java/com/wynntils/models/activities/ContentBookQueries.java

```diff
@@ -272,15 +272,15 @@ protected void toggleTracking(String name, ContentType contentType) {
         query.executeQuery();
     }
 
-    private int findTrackedContent(ContainerContent container, String name, ContentType contentType) {
+    private int findTrackedContent(ContainerContent container, String name, ActivityType activityType) {
```

Comment:
Content is actually rooted a bit deeper than just types.

Reply:
This is sooo weird! I searched for "content" in IntelliJ, and it does not show this! I have checked again and again now, it shows this if you open in the Find window, but not in the normal list. I have not selected any case or complete word filters. It just ignores the entire ContentBookQueries.java file.